src/iceberg/manifest/manifest_reader.cc

```diff
@@ -557,9 +558,9 @@ Result<std::vector<ManifestEntry>> ParseManifestEntry(
   return manifest_entries;
 }
 
-const std::vector<std::string> kStatsColumns = {"value_counts",     "null_value_counts",
-                                                "nan_value_counts", "lower_bounds",
-                                                "upper_bounds",     "record_count"};
+const std::vector<std::string> kStatsColumns = {
+    "value_counts", "null_value_counts", "nan_value_counts", "lower_bounds",
+    "upper_bounds", "column_sizes",      "record_count"};
```

Comment:
The newly added `column_sizes` seems not aligned with java impl, is that a bug in java or our implementation is different?